packages/web/src/components/track/GiantTrackTileProgressInfo.tsx

```diff
@@ -59,7 +59,7 @@ export const GiantTrackTileProgressInfo = ({
         </div>
       )
     }
-  } else {
-    return <p className={styles.progressText}>{messages.unplayed}</p>
   }
+
+  return <p className={styles.progressText}>{messages.unplayed}</p>
```

Comment:
This might seem weird. TS was complaining that this component couldn't be used in JSX because it might return `undefined`. Lo and behold, it might!

The way the conditionals were structured, there's a rare chance we would return nothing from this function (though it shouldn't be likely), which would be an error in React.